gravityflow/gravityflow
# Only enqueue theme assets for next bundle when shortcode found

## Description
This pr moves the enqueueing of the new theme js and css added recently to only occur for the theme when the shortcode is present.

## Testing instructions
* Load up a test site and confirm on non gravity flow enabled pages the theme js and css do not load
* Confirm on any shortcode enabled pages the files do load. 
 
## Screenshots
![image](https://user-images.githubusercontent.com/1568589/126077001-728d6fd2-aad2-4a0c-9fef-917be7f25ac5.png)
![image](https://user-images.githubusercontent.com/1568589/126077010-f1fcde62-7151-4ecc-9326-db2b95aa4d57.png)

## Checklist:
